base/abstractarray.jl

```diff
@@ -176,11 +176,11 @@ end
 ## Bounds-checking without errors ##
 function checkbounds(::Type{Bool}, sz::Dims, I...)
     n = length(I)
-    for dim = 1:(n-1)
+    for dim = 1:(n-1) #Fixme iter
```

Comment:
Here is a tuple containing the additional arguments (varargs). So it's fine to keep the current code. Same for `sz`, which is of type `Dims`, i.e. `Tuple{Vararg{Int64}}`.